### PR TITLE
[MIG] sale_cancel_reason: Migration to 12.0

### PR DESCRIPTION
**Pull request by victor-ditay**
cc @Tecnativa

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
Tested in runbot :+1: 

Just missing IMHO "Reason of cancelling" in the sale.report

http://3363449-818-2e20a6.runbot1.odoo-community.org/web#action=309&model=sale.report&view_type=pivot&menu_id=174

File: sale_cancel_reason/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to ...
```
Comment (pedrobaeza) [inline sale_cancel_reason/readme/USAGE.rst]: This is half-completed

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please squash some adjacent commits from bots

File: sale_cancel_reason/readme/USAGE.rst (hunk 1 above)
Comment (victor-ditay) [reply sale_cancel_reason/readme/USAGE.rst]: Changes done

**Review by pedrobaeza — APPROVED** [review]

File: sale_cancel_reason/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    'name': 'Sale Cancel Reason',
+    'version': '12.0.1.0.0',
+    'author': 'Camptocamp,'
+              'Tecnativa,'
+              'Odoo Community Association (OCA)',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'complexity': 'normal',
+    'website': "http://www.camptocamp.com",
```
Comment (nikul-serpentcs) [inline sale_cancel_reason/__manifest__.py]: https://github.com/OCA/sale-workflow

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

File: sale_cancel_reason/__manifest__.py [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
-          'data/sale_order_cancel_reason.xml',
-          ],
- 'auto_install': False,
- 'installable': True,
- }
+{
+    'name': 'Sale Cancel Reason',
+    'version': '12.0.1.0.0',
+    'author': 'Camptocamp,'
+              'Tecnativa,'
```
Comment (rousseldenis) [inline sale_cancel_reason/__manifest__.py]: @victormmtorres Maybe 'just' a migration does not make you as 'author'

Comment (pedrobaeza) [reply sale_cancel_reason/__manifest__.py]: I agree in this case. Sorry for not seeing it. Please only add you in contributors.

Comment (victor-ditay) [reply sale_cancel_reason/__manifest__.py]: Changes done

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**